modules/programs/emacs.nix

```diff
@@ -23,6 +23,7 @@ in
   options = {
     programs.emacs = {
       enable = mkEnableOption "Emacs";
+      service = mkEnableOption "Emacs daemon systemd service";
```

Comment:
Why not make it `service.enable`? I thought this was the agreed-upon way to do it in nixos.

Reply:
mbsync has 2 definitions, one for the program, one for the service. You might want to imitate that.